expertise/create_dataset.py

```diff
@@ -256,6 +256,11 @@ def retrieve_expertise(self):
         use_email_ids = self.config.get('use_email_ids', False)
         group_ids = self.convert_to_list(self.config.get('match_group', []))
         reviewer_ids = self.convert_to_list(self.config.get('reviewer_ids', []))
+
+        # If there is more than 1 group, only take the first group
+        if len(group_ids) > 1:
```

Comment:
Yes, the `match_group` field can either be a string or an array. In this function, the string gets converted into an array with a single element in the `self.convert_to_list()` function.

Here, I only take the first group because the first group corresponds to members whose publications will be dumped into the `archives` folder. The second group corresponds to members whose publications will be dumped into `submissions.json`